library/patient.inc

```diff
@@ -1057,6 +1058,62 @@ function pdValueOrNull($key, $value) {
     return "'$value'";
   }
 }
+// Supported input date formats are:
+//   yyyy/mm/dd
+//This function is to prevent user from entering the future date
+function fixDOB($date) {
```

Comment:
This is good. But I would suggest to make one more change. Do not allow to get past date of lets say beyond 100 yrs(we can sort out a suitable date range). Like in present case, any date like : "1001-01-01" is accepted. So this should be prevented.